m/mysql-connector-j/mysql-connector-j_ubi_8.5.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash -e
+
+# -----------------------------------------------------------------------------
+#
+# Package       : mysql-connector-j
+# Version       : 8.0.28
+# Source repo	: https://github.com/mysql/mysql-connector-j.git
+# Tested on     : UBI 8.5
+# Language      : Java
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Vikas Kumar <kumar.vikas@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=mysql-connector-j
+#PACKAGE_VERSION is configurable can be passed as an argument.
+PACKAGE_VERSION=${1:-8.0.28}
+PACKAGE_URL=https://github.com/mysql/mysql-connector-j.git
+
+yum install -y git wget java-1.8.0-openjdk-devel
+
+cd /root
+wget https://dlcdn.apache.org//ant/binaries/apache-ant-1.10.12-bin.tar.gz
+tar -xzf apache-ant-1.10.12-bin.tar.gz
+
+export PATH=/root/apache-ant-1.10.12/bin/:$PATH
+
+mkdir -p /root/libs
+cd /root/libs
+
+wget https://search.maven.org/remotecontent?filepath=org/junit/jupiter/junit-jupiter-engine/5.8.2/junit-jupiter-engine-5.8.2.jar -O junit-jupiter-engine-5.8.2.jar
```

Comment:
why do all these need to be installed outside of the main package and explicitly?

Reply:
This is as per the instructions provided at https://dev.mysql.com/doc/connector-j/8.0/en/connector-j-installing-source.html. Also added as comments in the script.